operator/pkg/ingress/annotations/annotations.go

```diff
@@ -16,6 +16,7 @@ const (
 	ServiceTypeAnnotation      = annotation.IngressPrefix + "/service-type"
 	InsecureNodePortAnnotation = annotation.IngressPrefix + "/insecure-node-port"
 	SecureNodePortAnnotation   = annotation.IngressPrefix + "/secure-node-port"
+	SSLPassthroughAnnotation   = annotation.IngressPrefix + "/tls-passthrough"
```

Comment:
Any reason we still call this `SSLPassthroughAnnotation`?

Reply:
Oops, renamed.